extensions/guacamole-auth-totp/src/main/java/org/apache/guacamole/auth/totp/conf/ConfigurationService.java

```diff
@@ -144,6 +156,22 @@ public int getPeriod() throws GuacamoleException {
         return environment.getProperty(TOTP_PERIOD, 30);
     }
 
+    /**
+     * Returns the maximun allowed offset between gucamole  and  TOTP token, in
+     * periodes. If not specified, 0 will be used by default.
```

Comment:
Is periodes the same as seconds?

Reply:
A period is the time interval where a particular code is valid, usually 30 seconds.

So an offset of 1 will accept the current code, the previous code and the following code.